core/templates/domain/summary/exploration-summary-backend-api.service.ts

```diff
@@ -60,9 +60,9 @@ export class ExplorationSummaryBackendApiService {
       explorationIds: string[],
       includePrivateExplorations: boolean,
       successCallback: (
-        value?: ExplorationSummaryBackendDict |
+        value: ExplorationSummaryBackendDict |
         Promise<ExplorationSummaryBackendDict>) => void,
```

Comment:
Can you create a type variable for this?

Reply:
updated